connector/connect/server/pom.xml

```diff
@@ -199,6 +199,12 @@
       <version>${tomcat.annotations.api.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
Why the move?

Reply:
It seems the server can't find the jar from class path.
![image](https://user-images.githubusercontent.com/8486025/223889202-4a56911e-336b-4695-ba40-8d052e2e52a1.png)